drivers/npmx/npmx_shell.c

```diff
@@ -936,6 +936,68 @@ static int cmd_buck_gpio_forced_pwm_set(const struct shell *shell, size_t argc,
 	return buck_gpio_set(shell, argc, argv, npmx_buck_forced_pwm_gpio_config_set);
 }
 
+static int cmd_buck_mode_set(const struct shell *shell, size_t argc, char **argv)
+{
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	if (argc < 2) {
+		shell_error(shell, "Error: missing buck instance index and buck mode.");
+		return 0;
+	}
+
+	if (argc < 3) {
+		shell_error(shell, "Error: missing buck mode.");
+		return 0;
+	}
+
+	int err = 0;
+	uint8_t buck_indx = (uint8_t)shell_strtoul(argv[1], 0, &err);
```

Comment:
256 will be a valid value here, please take a look at this -> https://github.com/nrfconnect/npmx/pull/112

Reply:
Or just let me know when it will be merged - I will align it within mine PR